src/hooks/useDailyStatistics/utils.ts

```diff
@@ -41,10 +44,12 @@ export const countTotalTransactionsAndByCategory = (
           category: key,
           quantityText: formatQuantityText(
             totalQuantityInCategory,
-            findItemByCategory?.quantity.unit || {
-              type: "POSTFIX",
-              label: " qty",
-            }
+            findItemByCategory?.quantity.unit
+              ? c13ntForUnit(findItemByCategory?.quantity.unit)
+              : {
+                  type: "POSTFIX",
+                  label: ` ${i18nt("checkoutSuccessScreen", "quantity")}`,
```

Comment:
Since this is not the checkout success screen, let's add this quantity key-value pair to the `statisticsScreen` object instead?

```
{
  ...
  checkoutSuccessScreen: {..., quantity: "qty"},
  statisticsScreen: {..., quantity: "qty"},
  ...
}
```

Reply:
updated!